src/DynamoCore/Graph/Workspaces/HomeWorkspaceModel.cs

```diff
@@ -195,6 +202,21 @@ internal virtual void OnSetNodeDeltaState(DeltaComputeStateEventArgs e)
                 verboseLogging,
                 isTestMode) { }
 
+        public HomeWorkspaceModel(Guid guid, EngineController engine,
```

Comment:
API Change : A new constructor is added for `HomeWorkspaceModel` to facilitate deserialization.